website/versioned_docs/version-v3.17.x/opa-versions.md

```diff
@@ -7,6 +7,17 @@ Gatekeeper depends on [Open Policy Agent](https://www.openpolicyagent.org/). To
 
 | Gatekeeper Version | OPA Version |
 | ------------------ | ----------- |
+| ------------------ | ----------- |
+| `v3.17.0`          | `v0.67.0`   |
+| `v3.16.3`          | `v0.64.1`   |
+| `v3.16.2`          | `v0.63.0`   |
+| `v3.16.0`          | `v0.60.0`   |
+| `v3.15.1`          | `v0.60.0`   |
+| `v3.15.0`          | `v0.60.0`   |
+| `v3.14.0`          | `v0.57.1`   |
+| `v3.13.0`          | `v0.54.0`   |
+| `v3.12.0`          | `v0.49.2`   |
+| `v3.11.1`          | `v0.47.2`   |
```

Comment:
only need to add versions after v3.15.1

Reply:
@ritazh Updated the versions. I also added a script to automate this process. Because of the script you will see there are tags and opa version added in the file that are older than 3.15.1.